scripts/common.py

```diff
@@ -712,3 +715,99 @@ def __enter__(self):
 
     def __exit__(self, etype, value, traceback):
         os.chdir(self.saved_path)
+
+
+@contextlib.contextmanager
+def managed_process(command_args, shell=False, timeout_secs=60, **kwargs):
+    """Context manager for starting and stopping a process gracefully.
+
+    Args:
+        command_args: list(int | str). A sequence of program arguments, where
+            the program to execute is the first item.
```

Comment:
Add: "Ints are allowed in order to accommodate e.g. port numbers."

Reply:
Done.